index.css

```diff
@@ -42,50 +47,152 @@ body {
+  width: 98%;
   display: flex;
-  justify-content: space-between;
-  padding-left: 21px;
-  padding-top: 16px;
-  padding-right: 21px;
+  justify-content: space-around;
+  background-color: #3e3c3c;
 }
 
-.desktop-navigation-bar {
-  display: none;
+.time-bar.active {
+  position: fixed;
+
+  /* display: flex;
+  justify-content: space-around; */
+
+  /* gap: 60%; */
+  width: 100%;
+  background-color: #3e3c3c;
+  padding: 10px;
+  top: 0;
 }
 
-.time-status {
-  color: var(--color-4100, #fff);
-  text-align: center;
-  font-family: 'Roboto', sans-serif;
-  font-size: 14px;
-  font-style: normal;
-  font-weight: 500;
-  line-height: normal;
-  letter-spacing: -0.28px;
+.menu-container {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 12px;
+  padding-bottom: 5px;
 }
 
-header {
+.menu-container.active {
+  position: fixed;
   background-color: #3c3a39;
+  width: 100%;
+  top: 20px;
 }
 
-.top-navigation {
-  width: auto 375px;
-  height: auto 48px;
-  display: flex;
-  justify-content: space-between;
+.nav-mail-icon {
+  display: none;
 }
 
-.top-navigation > a {
-  margin: 8px;
-  font-family: Roboto, sans-serif;
+.my-logo,
+.nav-link,
+.nav-menu {
   text-decoration: none;
-  padding: 10px 12px;
+  list-style: none;
+}
+
+.my-logo {
   color: var(--theme-3-neutral-11, #fff);
   text-align: center;
+  font-family: Roboto, sans-serif;
   font-size: 18px;
   font-style: normal;
   font-weight: 700;
-  line-height: 24px;
+  line-height: 133.333%;
+  padding: 10px 12px;
+  margin: 2px auto 2px 12px;
+}
+
+.menu-container.active .my-logo {
+  display: none;
+}
+
+.menu-container a {
+  color: white;
+}
+
+.nav-menu {
+  /* display: flex;
+  position: fixed; */
+
+  /* left: -100%; */
+  display: none;
+}
+
+.hamburger {
+  display: block;
+  cursor: pointer;
+  width: 24px;
+  height: 24px;
+  padding: 8px;
+  margin: 4px 24px 4px auto;
+}
+
+.bar {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: #fff;
+}
+
+.hamburger.active {
+  margin-left: 87%;
+}
+
+.hamburger.active .bar:nth-child(2) {
+  opacity: 0;
+}
+
+.hamburger.active .bar:nth-child(1) {
+  transform: translateY(8px) rotate(45deg);
+  background-color: #979493;
+}
+
+.hamburger.active .bar:nth-child(3) {
+  transform: translateY(-8px) rotate(-45deg);
+  background-color: #979493;
+}
+
+.nav-menu.active {
+  display: block;
+  position: fixed;
+  transition: 0.3s ease-in-out;
+  background-color: #3c3a39;
+  width: 100%;
+  height: 100%;
+  top: 4.5%;
+}
+
+.nav-menu.active .nav-link {
+  color: #fff5e1;
+  font-family: Inter, sans-serif;
+  font-size: 32px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 44px; /* 137.5% */
+  padding: 15px;
+}
+
+.nav-menu.active .nav-item {
+  border-bottom: 1px solid #6f6c6b;
+  width: 80%;
+  margin: 24px 24px auto 0;
+}
+
+.nav-mail-item.active {
+  visibility: hidden !important;
+}
```

Comment:
I think it would be better if you add some margin -top to the menu option so that the close button is not on the same line with the menu options  for a better UI 👍 

Your design :
![image](https://github.com/prince-rutagengwa/mobile-portfolio/assets/20537712/8792e5a5-93f6-4531-b337-85d1a6b76b9b)

Figma design :
![image](https://github.com/prince-rutagengwa/mobile-portfolio/assets/20537712/25053a9b-28c7-4cbf-8844-5d6d7df44584)